.eslintrc.js

```diff
@@ -7,4 +7,5 @@ module.exports = createConfig('eslint', {
     'global-require': 'off',
     'no-template-curly-in-string': 'off',
   },
+  ignorePatterns: ['example/**/*'],
```

Comment:
[inform] This root `.eslintrc.js` is no longer responsible for linting the `example` MFE app containing in this project. Instead, the `example` MFE app should be responsible for its linting config, given it has its own `.eslintrc.js` config file.

The root `lint` NPM script is modified below to ensure CI runs the `lint` command for both the root and the example MFE app.